src/core/middlewares/config.go

```diff
@@ -26,5 +27,8 @@ const (
 	COUNTQUOTA       = "countquota"
 )
 
+// ChartMiddlewares middlewares for chart server
+var ChartMiddlewares = []string{CHART}
```

Comment:
Where is the middleware chain initialized for chart?

Reply:
in the chart proxy.

Reply:
Sorry, fixed in `chart_repository.go` now